lib/stripe/recipient_transfer.rb

```diff
@@ -0,0 +1,23 @@
+module Stripe
+  class RecipientTransfer < APIResource
+    def self.delete(_, _, _)
+      raise NotImplementedError.new('RecipientTransfers cannot be updated.')
+    end
+
+    def self.list(_, _, _)
+      raise NotImplementedError.new('RecipientTransfers cannot be listed.')
+    end
+
+    def self.retrieve(_, _, _)
+      raise NotImplementedError.new('RecipientTransfers cannot be retrieved.')
+    end
+
+    def self.save(_, _, _)
+      raise NotImplementedError.new('RecipientTransfers cannot be saved.')
+    end
+
+    def self.update(_, _, _)
+      raise NotImplementedError.new('RecipientTransfers cannot be updated.')
+    end
```

Comment:
@remi-stripe I don't think you need any of these `NotImplementedError`s. These methods all come in only when you include certain API operation modules like this:

``` ruby
  class Account < APIResource
    extend Stripe::APIOperations::Create
    extend Stripe::APIOperations::List
    include Stripe::APIOperations::Delete
    include Stripe::APIOperations::Save
```